optimum_benchmark/backends/llama_cpp/backend.py

```diff
@@ -0,0 +1,93 @@
+from tempfile import TemporaryDirectory
+from typing import Any, Dict, Tuple
+
+from llama_cpp import Llama
+
+from optimum_benchmark.backends.base import Backend
+
+from .config import LlamaCppConfig
```

Comment:
```suggestion
from ..base import Backend
from .config import LlamaCppConfig
```
Let's not mix absolute and relative imports, it can lead to errors that are pretty annoying to debug. We usually prefer using relative imports in the source code.